### PR TITLE
Read census divisions and locations of service providers into db

### PR DESCRIPTION
**Pull request by Ruxuan**
Sample query:
```
SELECT * FROM service_providers
JOIN sp_locations ON service_providers.id=sp_locations.sp_id
WHERE sp_locations.main=true
```

### CODE REVIEW

File: scripts/setup_db.py [hunk 1]
```diff
@@ -33,45 +37,39 @@ def create_tables(hostname, dbname, user):
...
-                id INTEGER PRIMARY KEY,
-                census_division_id INTEGER REFERENCES census_division (id),
-                name TEXT NOT NULL,
-                website TEXT,
-                report_year INTEGER,
-                report_link TEXT,
-                expenses INTEGER,
+                id           INTEGER PRIMARY KEY,
+                name         TEXT NOT NULL,
+                website      TEXT,
```
Comment (silastsui) [inline scripts/setup_db.py]: should this be `NOT NULL`? is there value in having service providers that C4K can't look into

File: scripts/setup_db.py [hunk 2]
```diff
@@ -33,45 +37,39 @@ def create_tables(hostname, dbname, user):
...
+            """
+            CREATE TABLE sp_locations (
+                id        INTEGER PRIMARY KEY,
+                sp_id     INTEGER REFERENCES service_providers (id),
+                address   TEXT,
                 longitude DECIMAL(16,14),
-                latitude DECIMAL(16,14),
-                notes TEXT
+                latitude  DECIMAL(16,14),
+                main      BOOLEAN
```
Comment (silastsui) [inline scripts/setup_db.py]: what is this column used for? the name is a bit ambiguous

File: scripts/setup_db.py [hunk 3]
```diff
@@ -33,45 +37,39 @@ def create_tables(hostname, dbname, user):
...
-    except psycopg2.DatabaseError as e:
-        if con:
-            con.rollback()
-        print("Error {}".format(e))
-        sys.exit(1)
-
-    if con:
-        con.close()
+    for command in commands:
+        db.execute(command)
```
Comment (silastsui) [inline scripts/setup_db.py]: nice job cleaning this up ⭐

should `db.py` be in this PR? i don't see this method on master branch

File: scripts/setup_db.py (hunk 2 above)
Comment (Ruxuan) [reply scripts/setup_db.py]: It's a bool to indicate if the location is the service provider's main location.

File: scripts/setup_db.py (hunk 3 above)
Comment (Ruxuan) [reply scripts/setup_db.py]: oops yes it should be

**Pull request merged**